rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -226,6 +241,8 @@ class rocalTensor {
     unsigned copy_data(hipStream_t stream, void* host_memory, bool sync);
 #endif
     unsigned copy_data(void* user_buffer);
+    unsigned copy_data(void *user_buffer, uint last_batch_size);
+    unsigned copy_data(void* user_buffer, uint max_x1, uint max_y1);
```

Comment:
Why do we need 2 copy_data declarations for audio?

Reply:
removed